your-first-meeting.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: Attending your first PR Review Club Meeting
```

Comment:
perhaps omit "Meeting" here and line 6 below to be the same title as the home page link (or add "Meeting to the link text)